src/deletepy/operations/user_ops.py

```
@@ -0,0 +1,283 @@
+                                f"{RED}Warning: Connection info not available for user {user['user_id']}, skipping{RESET}"
+                            )
+                    else:
+                        # No connection filter, include all users
+                        user_ids.append(user["user_id"])
+
+            if user_ids:
+                return user_ids
+        return None
+    except ValueError as e:
+        print(f"{RED}Error parsing response for email {CYAN}{email}{RED}: {e}{RESET}")
+        return None
```

Pylint (3.3.7)

[refactor] Too many nested blocks (7/5)

(R1702)